### ISSUE TITLE
Default helm parameter name

### ISSUE THREAD

**Issue by LS80** [argoproj-labs/argocd-image-updater#211]
Is there a reason why the default helm parameter for the image name is `image.name` when it seems the convention appears to be `image.repository`? 

Some examples 
 - [Kubernetes Dashboard](https://github.com/kubernetes/dashboard/blob/e74d212cc9af8ce35702f88572edf29c5f68ae1f/aio/deploy/helm-chart/kubernetes-dashboard/templates/deployment.yaml#L60)
 - [Argo CD Notifications](https://github.com/argoproj/argo-helm/blob/f6069848f53cddb33f5be6e04b1eaebe824a3a5f/charts/argocd-applicationset/templates/deployment.yaml#L44)

I have so far been unable to find a helm chart in the wild using `image.name`.

**Comment by jannfis** [comment]
Hey @LS80, sorry for picking this up so late. 

The main reason is probably that I'm not much of a Helm user myself, and my knowledge about Helm parameter names and best practices around it are close to zero. :)

Do you feel this needs a change?

**Comment by LS80** [comment]
> Do you feel this needs a change?

Based on what I've seen in the wild, I would say yes 👍

**Comment by lebenitza** [comment]
Yes, please change this default behaviour to `image.repository`.

**Comment by LS80** [comment]
@jannfis Would a PR to change this be acceptable?
